src/datepicker/datepicker.module.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @name  patternfly datepicker
+ *
+ * @description
+ *   Select module for patternfly.
```

Comment:
Add description apropos for the control, eg. 

Datepicker module for patternfly

Reply:
nice catch. and sublime use of apropos.